components/ui/input.tsx

```diff
@@ -1,22 +1,33 @@
-import * as React from "react";
-
+import { forwardRef, type InputHTMLAttributes } from "react";
```

Comment:
was this from `pnpm dlx shadcn-ui@latest add input` ?

Reply:
because i customised the file (input already existed) i just removed the need to import the entirity of the react module even if it's pruned when compiling

Reply:
oh sorry i meant the changes to the file as a whole, were they changes you made manually or changes that came from updating via shadcn?

Reply:
Oh, yes i made these changes (Add the close button)

Reply:
i wonder if the move is to make a new ui component that wraps `Input`, something like `InputWithClear` ?